### ISSUE TITLE
[WebP] : Image file using the WebP filter loads with white background in Safari

### ISSUE THREAD

**Issue by jcastro-dotcms** [dotCMS/core#21652]
New versions of safari (14 and above) can render webp. We need to update our BinaryExporterServlet to respect the Safari version and only send a JPEG when it is < v14.

---

This was:

Reported via Support ticket: [106318](https://dotcms.zendesk.com/agent/tickets/106318)
Reproducible as of dotCMS 5.3.8.8, up to latest master (22.01)

Displaying a .PNG file on the an HTML Page in the front-end works as expected. However, if the `/webp/` filter is used, the very same image will display a white background in Safari.

**To Reproduce**
Please ask me for the specific image file to use for reproducing this problem:
1. Upload the image file provided by me.
2. Create an Advanced Template with the following data:
```html
<html>
    <header>
        <title>Test Template</title>
    </header>
    <body>
        <h1>This is a Test Template</h1>
        <div style='background-color: lightblue;'>
            <picture class="MenuItemDetail_hero__bg__2981O hidden lg:block self-end z-10 relative pointer-events-none">
                <source type="image/webp" srcset="https://demo.dotcms.com/{dA-LINK-TO-TEST-IMAGE}/600w/webp/75q">
                <img src="https://demo.dotcms.com/{dA-LINK-TO-TEST-IMAGE}/600w/webp/75q" alt="Alt text for this image">
            </picture>
        </div>
    </body>
</html>
```
3. Open up the page in the front-end using Firefox or Chrome. The image's background is transparent, which is expected.
4. However, in Safari, the image's background is blank.

**Expected behavior**
The WebP filter should work as expected in Safari as well.

**Additional context**
The only currently valid workaround is to remove the WebP filter. So, instead of doing this:
```
"https://demo.dotcms.com/{dA-LINK-TO-TEST-IMAGE}/600w/webp/75q"
```
Do this:
```
"https://demo.dotcms.com/{dA-LINK-TO-TEST-IMAGE}/600w"
```

**Comment by wezell** [comment]
This is because the quality filter is delivering a compressed JPEG if the user's browser is safari. Safari did not support webp until sometime in the past six months or so.

We can do a bit more logic that says something like `if browser is safari and safari version < 14` then deliver a jpeg, otherwise, deliver a webp image.

**Comment by dsilvam** [comment]
This is working as expected 
<img width="1648" alt="Screen Shot 2022-04-12 at 16 45 55" src="https://user-images.githubusercontent.com/1576794/163060136-328dbdc1-1623-4a96-bda7-9809e5bb2dc8.png">

**Comment by bryanboza** [comment]
Fixed, tested on release-22.06 // docker // FF 
This is working as expected
![image](https://user-images.githubusercontent.com/2641437/168874443-1a464147-0ccc-49aa-a0e1-d3512d045a0b.png)
